### DISCUSSION TITLE
repeatedly hitting ctrl shift s causes weird ui

### DISCUSSION THREAD

**Discussion by oeyoews** [TiddlyWiki/TiddlyWiki5#6739] [Q&A] [answered]
repeatedly pressing `ctrl shift L` causes a weird ui. Is it a bug?

![image](https://user-images.githubusercontent.com/72405338/175539406-159db9b2-dd9f-419d-9d8e-4565cef372c4.png)

**Comment by Jermolene** [comment, marked as answer]
Hi @oeyoews on the browsers I have tested it is repeatedly pressing <kbd>ctrl-shift-L</kbd> that has this effect. It is repeatedly displaying the same modal on top of itself; one can unwind them with the "close" button.

I think this qualifies as a bug. The underlying issue is that it is not possible for the keyboard handler to tell whether the layout switcher modal is already displayed. We could add some filter operators for reading the current modal stack.
